### DISCUSSION TITLE
Copilot nightly not working

### DISCUSSION THREAD

**Discussion by solafidei** [community/community#41813] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Hi, 

Copilot nightly version 1.63.7639 is no longer working ever since I updated VS Code to version 1.74. 
Pressing ctrl+enter does not bring up the suggestion window and the activate/deactivate button in the taskbar is missing. 

If it helps, the runtime status is "Not yet activated".

I dominantly use it in WSL but it does not work on the host machine either. 

I have tried various different combinations of installing/uninstalling/rebooting VS Code. 

Stable version 1.63.7601 is working fine so I can move to that for now but I wanted to make you guys aware of this.

**Comment by Rishikesh-Reddy** [comment]
Yeah, faced the same issue

Stable version is 1.63.7607 and It works on both WSL and my host machine

**Comment by Sid110307** [comment]
I am having the same issue. Version 1.63.7587 worked on WSL

**Comment by zxsleebu** [comment]
same for me.

**Comment by callumok2004** [comment]
+1

**Comment by WinkelCode** [comment]
Ever since 1.63.7639 it won't start with the error `[error] Error: Tests: either GH_COPILOT_TOKEN, or GITHUB_TOKEN, must be set, or there must be a GitHub token from an app with access to Copilot in undefined/.copilot-testing-gh-token. Run "npm run get_token" to get one.` 

You can click the cog icon and use "Install Another Version:" to roll back to 1.63.7607, which is the latest functional version.

**Comment by tonny8711** [comment]
+1

**Comment by awmorgan** [comment]
still broken in: 

Copilot Nightly:  1.63.7666 
Runtime status: Not yet activated.
VSCode: 1.74.1

confirmed rolling back to 1.63.7607 per @WinkelCode advice, and it works again

**Comment by DougTrajano** [comment]
Same for me

**Comment by mentasuave01** [comment]
Same for me

**Comment by whee** [comment]
This issue now seems to have hit the stable extension, which is at v1.64.7694.

**Comment by petercunha** [comment]
Same issue on latest stable version as of 16 December 2022

**Comment by ThioJoe** [comment]
Copilot was working an hour ago and suddenly stopped. It's as if it isn't activated at all.

Edit: You can work around it by installing the old version again. On the marketplace page for the extension, click the dropdown next to "Uninstall", hit "Install Another Version" and choose the 1.63.x version

**Reply by jaredlwong** [reply]
Yeah, just ran into this problem, downgrading to 1.63 fixed it

**Comment by slanterns** [comment]
![图片](https://user-images.githubusercontent.com/10987206/208221817-afcb73f9-c5cb-40ff-894e-9c4ab7c054ad.png)
![图片](https://user-images.githubusercontent.com/10987206/208221820-5ff6f569-abdb-4f28-8261-bef88dc366fc.png)
1.64.7694 stable is broken.

**Comment by bmuskalla** [comment, marked as answer]
We've just released `1.65.7705` to the marketplace which should hopefully resolve the issue all of you mentioned above. It may be necessary to reload VSCode after installing the upgrade in case you're still seeing authentication issues. We're really sorry for the inconvenience and thanks for all the timely feedback 🙇

**Reply by WinkelCode** [reply]
Nightly is still borked, I guess it will get the fix when the next build is released tonight?

**Reply by awmorgan** [reply]
@bmuskalla Nightly 1.63.7666 still broken

**Reply by awmorgan** [reply]
Nightly 1.65.7709 is working

**Reply by solafidei** [reply]
Thank you. 7709 is working

**Comment by pboudoin** [comment]
I'm on 1.65.7705 and I don't have any suggestion
